Core/src/Material/MaterialSlab.cpp

```diff
@@ -22,6 +23,9 @@ Acts::MaterialSlab::MaterialSlab(const Material& material, float thickness)
       m_thickness(thickness),
       m_thicknessInX0((eps < material.X0()) ? (thickness / material.X0()) : 0),
       m_thicknessInL0((eps < material.L0()) ? (thickness / material.L0()) : 0) {
+  if (thickness < 0) {
```

Comment:
not sure if you usually do such runtime checks in Acts but they were very useful for debugging this problem. I am a fan of fail fast

Reply:
Fine with me, consistency checking on construction is something I support, over-checking everything would certainly have a performance penalty.